server/src/test/proto/spine/test/entity/natural_number.proto

```diff
@@ -33,6 +33,7 @@ option java_multiple_files = true;
 // Zero is considered a natural number.
 //
 message NaturalNumber {
+    option (.entity).kind = ENTITY;
```

Comment:
Do we really need to have the `.` before the `entity`? I know this makes it topmost reference, but we normally don't do this in the type declarations. Why do it here?

Reply:
There is a [known issue](https://github.com/protocolbuffers/protobuf/issues/5223) in proto where package name clashes with the option name. In such cases we have to use the dot. 

For now I just renamed the test package, so it looks "cleaner".